### PR TITLE
Refactor chatbot workflow

### PR DESCRIPTION
**Pull request by lmilbaum**

### CODE REVIEW

File: .github/workflows/chatbot.yaml [hunk 1]
```diff
@@ -53,29 +54,25 @@ jobs:
...
         run: make functional_tests
 
-      - name: Login to github container registry
-        if: >
-          (github.event_name == 'push' || github.event_name == 'schedule') &&
-          (github.ref == 'refs/heads/main' || startsWith(github.ref, 'refs/tags/v'))
-        uses: docker/login-action@v3.1.0
+      - name: Login to Registry
+        if: github.event_name == 'push' && github.ref == 'refs/heads/main'
+        uses: redhat-actions/podman-login@v1.7
```
Comment (rhatdan) [inline .github/workflows/chatbot.yaml:67]: Should we have the name redhat in an upstream project?

Comment (lmilbaum) [reply .github/workflows/chatbot.yaml:67]: It is an open source GitHub Action https://github.com/redhat-actions/podman-login.

Comment (rhatdan) [reply .github/workflows/chatbot.yaml:67]: Ok, SGTM

**Comment by rhatdan** [comment]
LGTM

**Pull request merged**